commit 92ec5c93627eac0bbe3dfd7352b76733513d2afd
Author: John Crispin <john@phrozen.org>
Date:   Wed Jan 25 09:20:54 2017 +0100

    Documentation: devicetree: change the mediatek ethernet compatible string
    
    
    [ Upstream commit 61976fff20f92aceecc3670f6168bfc57a79e047 ]
    
    When the binding was defined, I was not aware that mt2701 was an earlier
    version of the SoC. For sake of consistency, the ethernet driver should
    use mt2701 inside the compat string as this is the earliest SoC with the
    ethernet core.
    
    The ethernet driver is currently of no real use until we finish and
    upstream the DSA driver. There are no users of this binding yet. It should
    be safe to fix this now before it is too late and we need to provide
    backward compatibility for the mt7623-eth compat string.
    
    Reported-by: Sean Wang <sean.wang@mediatek.com>
    Signed-off-by: John Crispin <john@phrozen.org>
    Reviewed-by: Matthias Brugger <matthias.bgg@gmail.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/Documentation/devicetree/bindings/net/mediatek-net.txt b/Documentation/devicetree/bindings/net/mediatek-net.txt
index c010faf..c7194e8 100644
--- a/Documentation/devicetree/bindings/net/mediatek-net.txt
+++ b/Documentation/devicetree/bindings/net/mediatek-net.txt
@@ -7,7 +7,7 @@ have dual GMAC each represented by a child node..
 * Ethernet controller node
 
 Required properties:
-- compatible: Should be "mediatek,mt7623-eth"
+- compatible: Should be "mediatek,mt2701-eth"
 - reg: Address and length of the register set for the device
 - interrupts: Should contain the three frame engines interrupts in numeric
 	order. These are fe_int0, fe_int1 and fe_int2.
